examples/default/src/screens/RepliesScreen.tsx

```diff
@@ -5,21 +5,72 @@ import { Heading } from 'native-base';
 
 import { ListTile } from '../components/ListTile';
 import { Screen } from '../components/Screen';
+import { Switch } from 'react-native';
 
 export const RepliesScreen: React.FC = () => {
   const [count, setCount] = useState<number>();
+  const [enableNotification, setEnableNotification] = useState<boolean>(true);
+  const [pushNotificationEnable, setPushNotificationsEnable] = useState<boolean>(true);
+  const [enableInAppReplayNotificationSound, setEnableInAppReplayNotificationSound] =
+    useState<boolean>(false);
+  const [enableSystemReplayNotificationSound, setEnableSystemReplayNotificationSound] =
+    useState<boolean>(false);
 
   useEffect(() => {
-    Replies.getUnreadRepliesCount(setCount);
+    Replies.getUnreadRepliesCount().then((unReadCount) => {
+      setCount(unReadCount);
+    });
   }, []);
 
   return (
     <Screen>
+      <ListTile title="Enable notification">
+        <Switch
+          value={enableNotification}
+          onValueChange={(value) => {
+            setEnableNotification(value);
+            Replies.setEnabled(value);
+          }}
+        />
+      </ListTile>
+      <ListTile title="InApp  Replay notification Sound">
+        <Switch
+          value={enableInAppReplayNotificationSound}
+          onValueChange={(value) => {
+            setEnableInAppReplayNotificationSound(value);
+            Replies.setInAppNotificationSound(value);
+          }}
+        />
+      </ListTile>
+      <ListTile title="System replay notification Sound">
+        <Switch
+          value={enableSystemReplayNotificationSound}
+          onValueChange={(value) => {
+            setEnableSystemReplayNotificationSound(value);
+            Replies.setSystemReplyNotificationSoundEnabledAndroid(value);
+          }}
+        />
+      </ListTile>
+      <ListTile title="Push notification enable">
```

Comment:
How about we rename this to `Push Notifications` to comply with the switch label naming conventions.